keychain/src/methods/register.ts

```diff
@@ -39,11 +34,23 @@ const register =
       "0",
     );
 
-    const controller = new Controller(keypair, address, credentialId);
```

Comment:
i think we should still keep this, since a controller can exist without being deployed. in some cases, the user will deploy it themselves